app/javascript/config/.manifest.json.d.ts

```diff
@@ -0,0 +1,6 @@
+interface Manifest {
+  [key: string]: string
+}
+
+declare const manifest: Manifest
+export default manifest
```

Comment:
Is there a reason this is `.manifest' with a leading `.` still. I feel like we could just make it `manifest.json` now

Reply:
We can. I'll update